opentech/apply/utils/blocks.py

```diff
@@ -76,7 +79,7 @@ def clean(self, value):
 
         if duplicates:
             all_errors.append(
-                'You have duplicates of the following required fields: {}'.format(', '.join(prettify_names(duplicates)))
+                'You have duplicates of the following non duplicate fields: {}'.format(', '.join(prettify_names(duplicates)))
```

Comment:
What about `The following fields must be included only once: {}`?

Reply:
that would work